### DISCUSSION TITLE
Date handling

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#151] [Q&A] [answered]
Hi, I'd like to open this discussion to capture questions about date handling, since it's easily the most complex part of a new contribution, depending of course on data quality on the target site.

I have to admit that I didn't know about the possibility of using regex before I submitted my first contribution. It would have helped.

Now for the question, is there any good way capture an hour with no minute part, and yet also capture time with both parts on the same website? Can I use two -capture stanzas? I'll try that, and if it works I'll answer my own question.

**Comment by MarkJaroski** [comment]
OK, so it's not possible to use the time component twice.

**Comment by MarkJaroski** [comment]
This is not an answer, but my own problem is solved: upon closer inspection I noticed a datetime attribute in the element containing the date. The attribute contains an ISO version of the datatime.

I should inspect their HTML for more semantic markup I suppose.

**Comment by jakopako** [comment]
I think this relates to [this issue](https://github.com/jakopako/goskyr/issues/168) in the goskyr project, right? Does this answer your question?

**Comment by MarkJaroski** [comment, marked as answer]
Got it! So the trick is to add two entries in the layout: array, like so:

```
            layout: ["Show: 15", "Show: 15.04"]
```
